.github/workflows/offlinedocs.yaml

```diff
@@ -0,0 +1,77 @@
+name: offlinedocs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "offlinedocs/**"
+  pull_request:
+    paths:
+      - "offlinedocs/**"
+  workflow_dispatch:
+
+# Cancel in-progress runs for pull requests when developers push
+# additional changes
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  fmt:
```

Comment:
TBH, all three of these should just be a single job that fmts, lints and builds offlinedocs to ensure it works. And it could probably go in the coder.yaml workflow as well.

Reply:
hmm. but we only want to run this when we chnage something in offlinedocs.
Why to lint and fmt it everytime. I could not think of a cleaner way to do this in ci.yaml

Reply:
Add a new section to the `changes` check called `offlinedocs`, then only run the `offlinedocs` check if it's true. And also add it to `required`.